addons/website/static/src/components/configurator/configurator.xml

```diff
@@ -182,12 +182,14 @@
                         </t>
                     </div>
                 </div>
-                <div class="text-right">
+                <div class="text-right pt-2 pt-lg-0">
                     <button class="btn btn-primary btn-lg ml-3" t-on-click="buildWebsite()">Build my website</button>
                 </div>
             </div>
+            <div class="ml-auto">
+                <SkipButton/>
+            </div>
         </div>
```

Comment:
@detrouxdev This commit replaces the issue by a worst one IMO. The point was to be able to access the "build my website" button at any time... now it is still not visible but the "skip" one is also invisible :shrug: 

Before:
![image](https://user-images.githubusercontent.com/10338094/143283161-7a73c4e7-7d8f-4f7e-8bc7-78e945cb067e.png)

With you commit:
![image](https://user-images.githubusercontent.com/10338094/143283261-7b0aeec3-16bd-42d1-b85a-fc4347b313cf.png)

The correct result can probably be achieve with less changes to the templates too (no need to change the SkipButton)

Reply:
I initially moved the build my website button to have it with the skip button but my solution would have pushed the build button from its original place (close to the features) down at the bottom of the screen on large screens

Original :
![Screenshot 2021-11-25 at 09 37 40](https://user-images.githubusercontent.com/77747125/143407983-5f4e9026-25ea-4562-b4ef-a756eba5a313.png)
My first solution:
![Screenshot 2021-11-25 at 09 37 33](https://user-images.githubusercontent.com/77747125/143408032-3bd93899-2c60-4acb-bd8b-4bef639a0073.png)

I talked with PDE about what he'd prefer and after discussing with FP they decided that they just wanted both buttons to be scrollable part on low resolution screen (Less important on phone sized screen since we can't edit the website on phone ATM).

So from this :
![Screenshot 2021-11-25 at 09 47 29](https://user-images.githubusercontent.com/77747125/143409022-ea2c149b-6f23-4e7b-8e5f-cc98ee80f6bf.png)

To this :
![Screenshot 2021-11-25 at 09 47 32](https://user-images.githubusercontent.com/77747125/143409058-7b616b53-e135-495f-aa25-14a84e215f1d.png)

If there's a way to have the button both stay on screen at all time but also be near the features on larger screens while also keeping the Skip button at the bottom, I wasn't able to find it 😞